src/kbmod/work_unit.py

```diff
@@ -249,8 +316,14 @@ def from_dict(cls, workunit_dict):
         else:
             global_wcs = None
 
+        constituent_images = workunit_dict["constituent_images"]
+        heliocentric_distance = workunit_dict["heliocentric_distance"]
```

Comment:
Should these be optional for an unreprojected workunit (as above)?

Reply:
I guess my general preference would be to just be explicit when we create a serialized representation of a `WorkUnit` to just have the attribute in the dict or fits file as a null value as I think it's easier to maintain long term as it requires less state checking about what is and isn't available.

Reply:
Sounds good